modules/apps/forms-and-workflow/dynamic-data-lists/dynamic-data-lists-web/src/main/resources/META-INF/resources/view_records.jsp

```diff
@@ -87,7 +87,7 @@ recordSearchContainer.setOrderByType(ddlViewRecordsDisplayContext.getOrderByType
 <aui:nav-bar cssClass="collapse-basic-search" markupView="lexicon">
 	<c:if test="<%= ddlDisplayContext.isAdminPortlet() %>">
 		<aui:nav cssClass="navbar-nav">
-			<aui:nav-item label="<%= recordSet.getName(locale) %>" selected="<%= true %>" />
+			<aui:nav-item label="<%= HtmlUtil.escape(recordSet.getName(locale)) %>" selected="<%= true %>" />
```

Comment:
@rafaprax @neilking 

Correct, this should probably be fixed in the taglib instead.